spyderlib/plugins/runconfig.py

```diff
@@ -467,8 +474,15 @@ def setup_page(self):
         interpreter_layout.addWidget(self.dedicated_radio)
         interpreter_layout.addWidget(self.systerm_radio)
         
-        wdir_group = QGroupBox(_("Working directory"))
-        wdir_bg = QButtonGroup(wdir_group)
+        general_group = QGroupBox("General settings")
+        command_options_enabled = self.create_checkbox(_("Command line options:"),

```

Comment:
I don't think we should expose *Command line options* in Preferences because it doesn't make much sense to let people set command line options *globally*. I think it only makes sense for each file.

What do you think?